### DISCUSSION TITLE
Selling extra PCBs we obtain from JLCPCB

### DISCUSSION THREAD

**Discussion by Sir-Wash** [zapta/ble_stepper_motor_analyzer#8] [General]
Hello, first sorry if this isn't where this kind of question is supposed to be placed. Just wanted to ask if there would be any issues will me selling the extra PCBs that I don't plan on using. I am ordering them off JLCPCB and the minimum I can get is 5. Is there any issue with selling the remaining ones? Thank you!

**Comment by zapta** [comment]
Hi @Sir-Wash,

No problem at all. You can sell as many as you want, as is or modified, for whatever price you want.  

Regards,
Zapta

**Reply by Sir-Wash** [reply]
Thank you for your response. I always double check just to verify as I don't want to cause issues, as I know you put alot of work into this and don't want to harm any business you may have. Do you have any donation links so I can send you something after I sell the boards? Having trouble find anything for ya.

**Reply by zapta** [reply]
@Sir-Wash, I don't have a donation link. I made this project with the intention that people like you will make it available for the community so thank you for doing so.

As a side note, I learned recently that I can order from JLCPCB 5 boards but only 2 of them populated with the SMD service. Don't know if this is a new feature or if I just missed it, but I use it for early prototype, especially if some of the parts are expensive.

**Reply by Sir-Wash** [reply]
Copy. Thank you for all the support you have provided to the community! Can't wait to see this thing in action!

**Reply by Sir-Wash** [reply]
Just noticed that the PCB design I want to use has Vector3D's logo on it. Does that design also fall under the same distribution controls as the others or do I need to reach out to him to get permission to selling the extras?

**Comment by zapta** [comment]
Yes, please contact him. Vector3D made design changes and those changes are
his IPs. In addition he owns the copyright to the Vector3D logo.

I would think that at minimum you don't want to distribute with his Logo on
which he has copyright but please check with him.

Zapta.

**Reply by Sir-Wash** [reply]
Thank you I will reach out to verify if it's alright with him.

**Comment by Sir-Wash** [comment]
Hello, so I created a fork and redesigned the board myself to add the features are I wanted. Wanted to run it by you before I order them. The design can be found here: https://github.com/Sir-Wash/ble_stepper_motor_analyzer. Let me know if you have any issues with what I have done.

**Comment by zapta** [comment]
Hi Sir_Wash,

I cloned your depot on my computer. What did you change? I didn't notice
changes to the schematic and PCB. This is for example what I see in the PCB
editor

[[image: image.png]](https://i.imgur.com/fXtpJw4.png)

**Reply by Sir-Wash** [reply]
So the files should be here "kicad/JLCPCB/Sir_Wash_Redesign" sorry haven't changed much of it yet will be working on that soon.

**Reply by Sir-Wash** [reply]
Where you able to view the changes?

**Comment by zapta** [comment]
Hi,

I have your repository on my local disk. Didn't see any changes in the
Kicad schematic and PCB layout editors so I presume you just added a
version of production files under kicad/JLCPCB/Sir_Wash_Redesign.

I try to upload those files to JLCPCB with SMD service, (with the BOM and
placement files) but JLCPCB hangs in the component placement stage and
doesn't show the component placement online editor. Were you able to upload
the project to JLCPCB?

EDIT: I tried to upload another project to JLCPCB and it also didn't work so, JLCPCB is broken for me. I need to try another computer.

Z.

**Reply by Sir-Wash** [reply]
Here is a link to what I am seeing on JLCPCB I will also check to see if it's an issue with the files I have added to the repo. https://imgur.com/a/mQurs4g

**Reply by zapta** [reply]
I was able to upload the project to JLCPCB on my other computer and as far as I can tell it looks good. Of course I can't guarantee anything, and I presume that you didn't change the JLCPCB/LCSC part numbers in the bom.

FYI, the JTAG header J5 is required only if you plan to do firmware development and is not necessary for regular users (firmware loading is done via the USB connector) so you may consider to non populating it. It's not a problem if you populate it anyway except that it's slightly more difficult to access the button next to it  (not a big deal though).

![image](https://github.com/zapta/ble_stepper_motor_analyzer/assets/2184769/c47f1b67-1bec-4c80-b66d-15a94d98d388)

**Reply by Sir-Wash** [reply]
Thank you! I am currently working on a case for this so the button issue shouldn't be a problem. Probably going to reconfigure the parts more before I lock into something.
